mdx/granule_metadata_extractor/src/helpers/creators/utils/mdx.py

```diff
@@ -233,13 +234,13 @@ def process_file(self, s3uri, collection_dir):
             raise
 
     def process_collection(self, short_name, provider_path, bucket='ghrcw-private'):
-        collection_dir = f'{tempfile.gettempdir()}/{short_name}'
+        collection_dir = f'{Path.home()}/{short_name}'
```

Comment:
Writing to the user (ec2-user) home is fine for now; however, this may cause issues in the future for troubleshooting / collaboration when migrated to PIV auth.